Source/WebCore/animation/CSSNumberishTime.h

```diff
@@ -33,17 +33,23 @@ namespace WebCore {
 class CSSNumberishTime {
 public:
     CSSNumberishTime() = default;
+    WEBCORE_EXPORT CSSNumberishTime(std::optional<Seconds>, std::optional<double>);
```

Comment:
Does this constructor have to be written like this (and potentially take invalid combinations of arguments) because of the generated IPC deserializer?

Reply:
That is indeed the reason why. The implementation will `ASSERT()` if the input is incorrect. We could write custom encoding/decoding code for that purpose. I agree this is not ideal and I will look into improving this as a follow-up.